sources/packages/web/src/components/students/modals/ConfirmEditApplication.vue

```diff
@@ -21,22 +20,42 @@ import { useModalDialog } from "@/composables";
 import { defineComponent } from "vue";
 
 export default defineComponent({
+  props: {
+    beforeEdit: {
+      type: Boolean,
+      required: true,
```

Comment:
IMO this is a supplementary parameter to display content. So it can have a default value and can be marked as optional.

Reply:
updated